graspy/inference/latent_distribution_test.py

```diff
@@ -14,121 +14,144 @@
 
 import numpy as np
 
-from ..embed import AdjacencySpectralEmbed, select_dimension
+from ..embed import select_dimension, AdjacencySpectralEmbed
 from ..utils import import_graph, is_symmetric
 from .base import BaseInference
+from .dists import euclidean
+from hyppo.ksample import KSample
+from scipy.stats import multiscale_graphcorr
+from joblib import Parallel, delayed
 
 
 class LatentDistributionTest(BaseInference):
     """
-    Two-sample hypothesis test for the problem of determining whether two random 
+    Two-sample hypothesis test for the problem of determining whether two random
     dot product graphs have the same distributions of latent positions.
-    
+
     This test can operate on two graphs where there is no known matching between
-    the vertices of the two graphs, or even when the number of vertices is different. 
+    the vertices of the two graphs, or even when the number of vertices is different.
     Currently, testing is only supported for undirected graphs.
 
     Read more in the :ref:`tutorials <inference_tutorials>`
 
     Parameters
     ----------
+    test : str, one of ["CCA", "Dcorr", "HHG", "RV", "Hsic", "MGC"]
+        Independence test to use.
+
+    distance : function, (default=`function <euclidean>`)
```

Comment:
just to be consistent with sklearn, they usually call this `metric` if it is a distance

Reply:
also i think this should accept strings by default, with an option to pass functions if you explain what they take (and think it would be useful for the user to define their own functions)

Reply:
right